jmix-ui/ui-data/src/main/java/io/jmix/uidata/action/filter/FilterSaveAsAction.java

```diff
@@ -185,6 +186,14 @@ protected void openInputDialog() {
                                 "logicalFilterConditionEdit.uniqueConfigurationId"));
                     }
 
+                    String name = validationContext.getValue("nameField");

```

Comment:
What will happen if I mark my personal config as global?

Reply:
Added separate validations for user and global configurations